lib/fluent/plugin/out_nsq.rb

```diff
@@ -1,23 +1,25 @@
 # coding: utf-8
 
-module Fluent
-  class NSQOutput < BufferedOutput
-    Plugin.register_output('nsq', self)
+require 'fluent/plugin/output'
+require 'nsq'
+require 'yajl'
+
+module Fluent::Plugin
+  class NSQOutput < Output
+    Fluent::Plugin.register_output('nsq', self)
 
     config_param :topic, :string, default: nil
     config_param :nsqlookupd, :string, default: nil
```

Comment:
Can we use `array` type instead of `string`?

Reply:
OK. I've modified the type to array (also the same fix has been applied to `in_nsq`)

See 7c6ef1d for the actual diff.